src/sound/mod.rs

```diff
@@ -554,9 +565,12 @@ impl SoundManager {
 pub(in crate::sound) enum PlaybackMessage {
     Stop,
     PlayOrganyaSong(Box<Song>),
+    PlayOrganyaSongNoLoop(Box<Song>),
     #[cfg(feature = "ogg-playback")]
     PlayOggSongSinglePart(Box<OggStreamReader<File>>),
     #[cfg(feature = "ogg-playback")]
+    PlayOggSongSinglePartNoLoop(Box<OggStreamReader<File>>),
```

Comment:
we should not duplicate any of that

we should add a `SongParameters` struct instead and pass it into this enum